website/src/components/token/Token.js

```diff
@@ -51,24 +51,43 @@ export default function Token({
 
   const hasDisplay = !!(configuration?.displayName && hasValue && isToken);
 
-  const emptyConfiguration = Object.keys(configuration || {}).length === 0;
+  const hasEmptyConfig = !Object.keys(configuration || {}).length;
 
-  const _errorToken = !selectedToken?.token && "😬 This is not a valid token!";
+  // Validation status class'
+  const isExpired = hasEmptyConfig && status === 404;
+  const isInvalid = hasEmptyConfig && status === 401;
+  const isNotVerified = hasEmptyConfig && status === 500;
+  const isError = hasEmptyConfig && status !== 200;
+
+  const hasStatusError = isExpired || isInvalid || isNotVerified || isError;
+
```

Comment:
Kan se at den samme logik også bruges i History.page.js. Kunne man ikke samle det til en funktion der tager i mod hasEmptyConfig og status og så returnerer den et objekt der har alle de felter? 
Så noget i den stil:

`const {isExpired, isInvalid, isError } = getValidation(hasEmptyConfig , status );`

Reply:
Jo jeg ville egentlig gerne have det helt ind i hook'en, så den bare smed en af de ovenstående status'er. Men jeg vurderede at det ville tage for meget ekstra tid - mest fordi det ikke er noget jeg har fået lov at lave :D 

Men det kunne da godt være jeg lige skulle give det et skud :thinking: